loadtest/runner/load_test_runner_test.go

```diff
@@ -0,0 +1,40 @@
+package runner
+
+import (
+	"testing"
+	"time"
+
+	"github.com/0xPolygon/polygon-edge/crypto"
+	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/stretchr/testify/require"
+	"github.com/umbracle/ethgo/wallet"
+)
+
+func TestMnemonic(t *testing.T) {
+	realAddress := types.StringToAddress("0x85da99c8a7c2c95964c8efd687e95e632fc533d6")
+
+	key, _ := wallet.NewWalletFromMnemonic("code code code code code code code code code code code quality")
+	raw, _ := key.MarshallPrivateKey()
+
+	ecdsaKey, _ := crypto.NewECDSAKeyFromRawPrivECDSA(raw)
+	require.Equal(t, realAddress, ecdsaKey.Address())
+}
+
+func TestLoadRunner(t *testing.T) {
```

Comment:
Hm maybe it is better to create a README with instructions on how to run it?

Reply:
I left this only for purposes of debugging the tool if needed.